docs/remote/containers.md

```diff
@@ -264,18 +278,16 @@ Editing your container configuration is easy. Since rebuilding a container will
 
 Here is the typical edit loop using these commands:
 
+![Container edit loop illustration](images/containers/container-edit-loop.png)
+
 1. Start with **Remote-Containers: Add Development Container Configuration Files...** in the Command Palette (`kbstyle(F1)`).
 2. Edit the contents of the `.devcontainer` folder as required.
 3. Try it with **Remote-Containers: Reopen Folder in Container**.
-4. On failure:
-   1. **Remote-Containers: Reopen Folder Locally**, which will open a new local window.
-   2. In this local window: Edit the contents of the `.devcontainer` folder as required.
-   3. Try it again: Go back to the container window, **Developer: Reload Window** from the Command Palette (`kbstyle(F1)`)
-   4. Repeat as needed.
-5. If the build was successful, but you want to make more changes:
-      1. Edit the contents of the `.devcontainer` folder as required when connected to the container.
-      2. **Remote-Containers: Rebuild Container**.
-      3. On failure: Follow the same workflow above.
+4. If you see an error, click on **Open Folder Locally** in the dialog that appears.
+5. A copy of the **build log will appear** in the new, local window so you can investigate the problem. Edit the contents of the `.devcontainer` folder as required.
```

Comment:
Detail: It's not a "new" window anymore, the window is reused. That should make handling the window(s) easier. It also required a way to get at the old log. In addition to opening the log automatically (which might not always be the right thing to do...), there is also the command **Remote-Containers: Open Log File...**.